### PR TITLE
Update to the latest rc

### PR DESCRIPTION
**Pull request by jtpio**
Run the upgrade script to update to the latest RC:

`python -m jupyterlab.upgrade_extension`

cc @jasongrout since this was mentioned yesterday during the BOF at JupyterCon

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -25,29 +25,31 @@
...
-    "@jupyterlab/application": "^3.0.0-alpha.14",
-    "@jupyterlab/apputils": "^3.0.0-alpha.14",
+    "@jupyterlab/application": "^3.0.0-rc.5",
+    "@jupyterlab/apputils": "^3.0.0-rc.5",
     "@lumino/messaging": "^1.4.3",
     "@lumino/widgets": "^1.14.0"
   },
   "devDependencies": {
-    "@jupyterlab/builder": "^3.0.0-alpha.14",
+    "@jupyterlab/builder": "^3.0.0-rc.2",
```
Comment (jtpio) [inline package.json]: Interestingly the upgrade script did not update this one all the way up to `^3.0.0-rc.5`.

Comment (jtpio) [reply package.json]: Manually updated in https://github.com/jupyterlab/jupyterlab_apod/pull/8/commits/a93463d6aafb252beed5bbe5837a0c5cf0f0cedb for now.

**Comment by jasongrout** [comment]
Remember that this repo is special, in that every commit matters because they follow the steps in the guide. In order to update this repo, follow the guide at https://github.com/jupyterlab/jupyterlab/blob/master/RELEASE.md#updating-the-extension-tutorial

**Comment by jtpio** [comment]
Yes indeed.

The tutorial shouldn't need any update since this PR is mostly about updating dependencies, and doesn't mention the `package.orig.json` (hidden in `setup.py`).

**Comment by jasongrout** [comment]
> The tutorial shouldn't need any update since this PR is mostly about updating dependencies, and doesn't mention the `package.orig.json` (hidden in `setup.py`).

I agree. I think the simple rebase method should work fine.

**Comment by jasongrout** [comment]
I force-pushed an update to the 3.0 branch. Do you want to take a look to make sure it looks okay?

**Comment by jtpio** [comment]
Ah just realized that the progression (and git history) is actually part of the tutorial :)

Thanks for the fix!

**Pull request closed without merging**